source/source_file.py

```diff
@@ -1201,6 +1212,23 @@ def optimize_inclusions(self, log = lambda *args, **kw : None):
                         substitution.origin.path
                     ))
 
+                    if graphs_prefix is not None:
+                        # Highlight chunks of interest.
+                        prev_color = substitution.appearance.get("color")
```

Comment:
Maybe use `fillcolor` to keep the black border of the node?
(note: `fillcolor` works only with style `filled`)

Reply:
This has pitfalls resulting in code sophistication, meanwhile it's not useful enough visually.